server/app.ts

```diff
@@ -8,15 +8,21 @@ import cors from 'cors';
 dotenv.config();
 
 const app = express();
-console.log(process.env.PORT);
 const port: number = process.env.PORT ? parseInt(process.env.PORT) : 1338; // development port is 1338
 const isDevelopment = process.env.NODE_ENV === 'test';
 
 // Use EJS as the template engine
 app.set('view engine', 'ejs');
 app.set('views', path.join(__dirname, 'views'));
 
-app.use(cors());
+app.use(
+  cors({
+    origin: [process.env.MAIN_URL as string],
```

Comment:
Will have to remember to add this variable to github actions

Reply:
yea, need to update the .env for both ui and server